### PR TITLE
add earrings selector

### PR DESCRIPTION
**Pull request by 20RaVen23**
Updated zz_selector.rpy for earrings sprites in #6681

### CODE REVIEW

**Comment by Rai99** [comment]
Is that a

**Comment by 20RaVen23** [comment]
...That's bad, right?

**Comment by Rai99** [comment]
You generally aim for `content`.
[The guide to rebranching can be found here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-put-my-pull-request-in-the-wrong-branch-and-dont-want-to-suffer-the-wrath-of-devs-how-do-i-fix-it)

**Comment by 20RaVen23** [comment]
Sorry about that...

**Comment by multimokia** [comment]
Going to close and reopen to jumpstart Travis

File: Monika After Story/game/zz_selector.rpy [hunk 1]
```diff
@@ -3818,5 +3823,26 @@ label monika_hat_select:
...
+            persistent.event_database,
+            eventlabel="monika_earrings_select",
+            category=["appearance"],
+            prompt=store.mas_selspr.get_prompt("earrings", "change"),
+            pool=True,
+            unlocked=False,
+            rules={"no_unlock": None},
+            aff_range=(mas_aff.HAPPY, None)
+        ),
+        restartBlacklist=True
```
Comment (multimokia) [inline Monika After Story/game/zz_selector.rpy:3838]:
```suggestion
        restartBlacklist=True,
```

**Review by multimokia — CHANGES_REQUESTED** [review]

File: Monika After Story/game/zz_selector.rpy [hunk 2]
```diff
@@ -3818,5 +3823,26 @@ label monika_hat_select:
...
+            pool=True,
+            unlocked=False,
+            rules={"no_unlock": None},
+            aff_range=(mas_aff.HAPPY, None)
+        ),
+        restartBlacklist=True,
+        markSeen=True
+    )
+
+label monika_earring_select:
```
Comment (multimokia) [inline Monika After Story/game/zz_selector.rpy:3842]:
```suggestion
label monika_earrings_select:
```

**Review by multimokia — CHANGES_REQUESTED** [review]

File: Monika After Story/game/zz_selector.rpy [hunk 3]
```diff
@@ -79,6 +79,10 @@ init -100 python in mas_selspr:
             "change": "Can you tie your hair with something else?",
             "wear": "Can you tie your hair with something else?",
         },
+        "_ev": "monika_earrings_select",
+        "_min-items": 1,
+        "change": "Can you change your earrings?",
+        "wear": "Can you wear earrings?",
```
Comment (ThePotatoGuy) [inline Monika After Story/game/zz_selector.rpy:85]:
```suggestion
        "earrings": {
            "_ev": "monika_earrings_select",
            "_min-items": 1,
            "change": "Can you change your earrings?",
            "wear": "Can you wear earrings?",
        },
```

can you also move this up so its in alphabetical order with the rest of the ACS?

**Review by ThePotatoGuy — CHANGES_REQUESTED** [review]

File: Monika After Story/game/zz_selector.rpy [hunk 4]
```diff
@@ -79,6 +79,12 @@ init -100 python in mas_selspr:
             "change": "Can you tie your hair with something else?",
             "wear": "Can you tie your hair with something else?",
         },
+        "earrings": {
```
Comment (ThePotatoGuy) [inline Monika After Story/game/zz_selector.rpy:82]: move this block up between `clothes` and `hair` so it is in alphabetical order.

**Review by ThePotatoGuy — CHANGES_REQUESTED** [review]

**Review by ThePotatoGuy — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Pull request merged**